docs/index.md

```diff
@@ -13,35 +17,35 @@ you are encouraged to contribute, share insights, and collaborate with fellow en
 
 - [License](LICENSE link here)
 - [Community guide](community_guide.md)
-  - [Code of Conduct](community_guide.md#code-of-conduct)
-  - [Communication](community_guide.md#communication)
-    - [Asynchronous channels](community_guide.md#asynchronous-channels)
-    - [Real-time channels](community_guide.md#real-time-channels)
+    - [Code of Conduct](community_guide.md#code-of-conduct)
+    - [Communication](community_guide.md#communication)
+        - [Asynchronous channels](community_guide.md#asynchronous-channels)
+        - [Real-time channels](community_guide.md#real-time-channels)
```

Comment:
I don't actually like this manually-created table of contents. It is difficult to keep updated and  is duplicated by the left-hand navigation. I'd rather see this entire section removed as redundant and difficult to maintain and not something we should suggest projects use.
@oraNod what do you think?

Reply:
Yeah, manual TOC is not great. Maybe it's my holiday brain but I thought the point here is to highlight the TOC as part of the template, not to actually create the TOC.

@samccann btw I enabled PR previews in the RTD server-side config because it doesn't show up for this PR. Maybe I'll close it and reopen to manually trigger the preview, which will make it easier to review.